src/Uno.UWP/Generated/3.0.0.0/Windows.Storage/StreamedFileFailureMode.cs

```diff
@@ -2,7 +2,7 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.Storage
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+	#if false
 	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
 	[global::Uno.NotImplemented]
 	#endif
```

Comment:
Are we actually supporting those modes or you just copied the enum into the _implemented_ folders?

Reply:
Essentially I just moved it, but for now we actually only support the fact the loading failed. All cases are handled the same way. But your question is valid and I'll add the `[NotImplemented]` attribute on other values.